### PR TITLE
Include Organisation in notification when assigning.

### PR DESCRIPTION
**Pull request by smn**
Currently it only shows the user, not the organisation the user is part of.

### CODE REVIEW

**Comment by smn** [comment]
@rudigiesler @KaitCrawford @jpaidoussi ready for review.

File: casepro/cases/models.py [hunk 1]
```diff
@@ -95,7 +95,11 @@ def release(self):
         self.save(update_fields=('is_active',))
 
     def as_json(self, full=True):
-        result = {'id': self.pk, 'name': self.name}
+        result = {
+            'id': self.pk,
+            'name': self.name,
+            'org': self.org.name,
```
Comment (jpaidoussi) [inline casepro/cases/models.py]: Does this attribute need to outside of full flag? They seem to prefer the non-full state to be just id and name.

**Comment by smn** [comment]
bleh those last two commits were meant to go into #114 instead.

**Comment by smn** [comment]
I'm closing this entire PR after having spoken to @rudigiesler I have a better idea on how to go about this one:

```
The issue here is firstly I think we want the partner, not the organization (organizations are more like having separate sites in one install). Secondly, we need to store the partner when the CaseAction gets created, because it could change later, and then the history is incorrect.
```

**Pull request closed without merging**